docs/Developers/Git_Basics.rst

```diff
@@ -245,6 +245,12 @@ the desired branch:
     $ git fetch origin
     $ git rebase origin/master
 
+.. note::
+
+    ``origin`` is the remote of https://github.com/coala/coala
+    or https://github.com/coala/coala-bears. It is not the remote
+    of your fork.
```

Comment:
The earliest I could trace the original file to is: https://github.com/coala/documentation/blob/e2a113921a27fc77ddb47c668e4db5fa20a8a0fc/Users/Tutorials/Git_Help.rst